lib/msal-react/src/utils/utilities.ts

```diff
@@ -22,3 +24,26 @@ export function getChildrenOrFunction<T>(
 type SetDifference<A, B> = A extends B ? never : A;
 type SetComplement<A, A1 extends A> = SetDifference<A, A1>;
 export type Subtract<T extends T1, T1 extends object> = Pick<T,SetComplement<keyof T, keyof T1>>;
+
+/**
+ * Helper function to determine whether 2 arrays are equal
+ * Used to avoid unnecessary state updates
+ * @param arrayA 
+ * @param arrayB 
+ */
+export function accountArraysAreEqual(arrayA: Array<AccountInfo>, arrayB: Array<AccountInfo>): boolean {
```

Comment:
Can you add tests for this?

Reply:
Done. Still want to figure out how to properly test that the component is rendering and rerendering when I expect it to, but I'll tackle that at some point before GA

Reply:
Sounds good!